### DISCUSSION TITLE
Is there a way to change the Pump status?

### DISCUSSION THREAD

**Discussion by ruilee16** [pyswmm/pyswmm#373] [Q&A]
In SWMM software, we can define if a pump object is initially ON or OFF. Is there a way to change the pump's initial status in pyswmm? Or is changing the .inp file the only way to turn off a pump?

**Comment by dokzai** [comment]
Hello @ruilee16

You can adjust the pump setting via [target_setting](https://pyswmm.readthedocs.io/en/stable/tutorial/tutorial.html?highlight=target_setting#links).
